### PR TITLE
Sudoku example v2

### PR DESCRIPTION
**Pull request by alex-cornejo**
* Contains an implementation of a GA for the sudoku.
* Contains a custom crossover.
* Provides four different boards for testing.
* Important sections of code are documented.
* A SudokuGrid class that abstracts logic of sudoku penalties

This is a PR v2 of https://github.com/jenetics/jenetics/pull/810

### CODE REVIEW

**Comment by alex-cornejo** [comment]
I am open to any suggestion aiming to improve this implementation :)
Thanks so much for the support.

File: jenetics.example/src/main/java/io/jenetics/example/sudoku/SudokuUtil.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * Provides grids (boards) examples for sudoku.
+ * Zeros represent empty cells that must be filled.
+ *
+ * @author José Alejandro Cornejo Acosta
+ */
+public class SudokuUtil {
+
+	public static final int SIZE = 9;
+
+	public static final int[][] BOARD1 = {
```
[the remark this answers is not in the document]
Comment (jenetics) [reply jenetics.example/src/main/java/io/jenetics/example/sudoku/SudokuUtil.java:32]: With a `Board` class it is also possible to do some precondition check and throw an `IllegalArgumentException` for invalid boards.

**Review by jenetics — CHANGES_REQUESTED** [review]
Thank you for the new version. Looks way better. I only have a view comments ;-)

**Review by jenetics — APPROVED** [review]
Locks good. Will merge into r7.0.0 branch.

**Pull request merged**